weave/engine_trace.py

```diff
@@ -155,23 +155,23 @@ def attributes(self):
     def set_tag(self, key, unredacted_val, redacted_val=None):
         if "tags" not in self.attributes:
             self.attributes["tags"] = {}
-        if not os.getenv("DISABLE_WEAVE_PII"):
+        if not os.getenv("DISABLE_WEAVE_PII") == "true":
```

Comment:
Nit: Maybe a `lower()` and a `strip()` call on the getenv would be a good idea for input safety.

Reply:
I think creating a helper method may be the way to go.

Reply:
yup making a helper in environment.py